src/main/java/org/prebid/server/currency/CurrencyConversionService.java

```diff
@@ -139,11 +140,9 @@ public BigDecimal convertCurrency(BigDecimal price, Map<String, Map<String, BigD
             return price;
         }
 
-        final BigDecimal conversionRate = usepbsrates
-                    ? getConversionRateFromServer(externalCurrencyRates, requestCurrencyRates, adServerCurrency,
-                    effectiveBidCurrency)
-                    : getConversionRateFromRequest(externalCurrencyRates, requestCurrencyRates,
-                    adServerCurrency, effectiveBidCurrency);
+        final BigDecimal conversionRate = BooleanUtils.isTrue(usepbsrates)
```

Comment:
`"usepbsrates": false // defaults to true `  -> null + true
`BooleanUtils.isTrue` // only true 
You need to use isFalse and reverse if statement or isNotFalse.